### PR TITLE
Converting all antag equip procs to outfits.

### PR DESCRIPTION
**Pull request by MistakeNot4892**

### CODE REVIEW

File: mods/species/ascent/datum/antagonist.dm [hunk 1]
```diff
@@ -35,13 +36,13 @@
...
+	else
+		rig_type = initial(rig_type)
 	. = ..()
 	if(.)
-		if(player.species.get_root_species_name(player) == SPECIES_MANTID_GYNE)
-			equip_rig(/obj/item/rig/mantid/gyne, player)
-		else
-			equip_rig(/obj/item/rig/mantid, player)
-			player.put_in_hands(new /obj/item/gun/energy/particle)
+		player.put_in_hands(new /obj/item/gun/energy/particle)
```
Comment (MistakeNot4892) [inline mods/species/ascent/datum/antagonist.dm:45]: This is technically not 1:1 but hunter is unplayable and I have plans to port the redone mode soon so w/e.

File: code/game/antagonist/station/traitor.dm [hunk 2]
```diff
@@ -64,48 +64,38 @@
...
-	spawn_uplink(traitor_mob)
-	give_intel(traitor_mob)
-
-/decl/special_role/traitor/proc/give_intel(mob/living/traitor_mob)
-	give_collaborators(traitor_mob)
-	give_codewords(traitor_mob)
+/decl/special_role/traitor/equip(var/mob/living/carbon/human/player)
+	. = ..()
+	if(.)
+		if(istype(player, /mob/living/silicon)) // this needs to be here because ..() returns false if the mob isn't human
```
Comment (comma) [inline code/game/antagonist/station/traitor.dm]: does this work? is comment lying?

Comment (MistakeNot4892) [reply code/game/antagonist/station/traitor.dm]: I have no idea. Shouldn't do.

Comment (MistakeNot4892) [reply code/game/antagonist/station/traitor.dm]: Misunderstood this comment, fixed now.

**Comment by MistakeNot4892** [comment]
Tested, all seems to be working.

**Review by comma — APPROVED** [review]

**Pull request merged**